tensorboard/webapp/metrics/views/right_pane/settings_view_component.ng.html

```diff
@@ -74,6 +74,22 @@ <h3 class="section-title">Scalars</h3>
       >Ignore outliers in chart scaling</mat-checkbox
     >
   </div>
+
+  <div class="control-row scalars-partition-x">
+    <mat-checkbox
+      [checked]="scalarPartitionX"
+      (change)="scalarPartitionXToggled.emit()"
+      >Partition non-monotonic X axis</mat-checkbox
+    >
+    <mat-icon
+      class="info"
+      svgIcon="help_outline_24px"
+      matTooltip="Non-monotonic steps can occur when reusing logdir with multiple summary
+      writers and overlapping step numbers. Line chart, without this option, can appear
+      zig zag. This is common when restarting from a checkpoint."
```

Comment:
Wording suggestions:
"can occur when reusing logdir with" --> "can occur when reusing a logdir with"
"Line chart, without this option, can appear zig zag." --> "Line charts, without this option enabled, can appear zig zagged."

I would also add a comment explaining what to expect after enabling the checkbox.  e.g. something like "When enabled, a non-monotonic time series composed of N monotonic pieces will be shown as N monotonic lines on the line chart".